### DISCUSSION TITLE
https://nanaeo.cn/friends/

### DISCUSSION THREAD

**Discussion by giscus** [MliKiowa/Mlikiowa.github.io#4] [General]
# https://nanaeo.cn/friends/

这里写友链上方的内容。 - Mlikiowa - Village

https://nanaeo.cn/friends/

**Comment by xxhzm** [comment]
网站名称：小小孩子们的Blog
网站链接：https://www.xxhzm.cn
网站图片:   https://cdn.xxhzm.cn/xxhzm/head.jpg
网站描述：记录美好生活

**Reply by MliKiowa** [reply]
已添加啦

**Comment by FreeDivers** [comment]
观光

**Comment by Mitpoppy** [comment]
网站名称：幻想の小窝
网站链接：https://www.someo.top
网站图片:  https://www.someo.top/favicon.ico
网站描述：心生向往，在厉前行

**Comment by wzdc** [comment]
有的网站把你取消友链了。。不管管嘛？

**Reply by MliKiowa** [reply]
这个嘛 其实可以理解 我只后检查检查就是啦

**Comment by leveljava** [comment]
名称:微乐的个人博客
介绍:“固所愿者，唯相伴也，固所求者，唯相守也。”
链接:https://sunalex.cn
头像:https://liwei-graduation.oss-cn-shenzhen.aliyuncs.com/aurora/avatar/29f3acebe23ae8520a874fac51c718b2.jpg

**Comment by galaxy2017** [comment]
您好，我是友链上的胖螺。
因为一些原因。站点已经从 blog.luomoe.com 迁移至 https://www.nahida.cn/
博客头像的地址更新为   https://www.nahida.cn/upload/logo.svg

**Reply by MliKiowa** [reply]
好的 随后更新嗷
